lib/common/splash/splash_screen.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+
+import '../../navigator/route_names.dart';
+
+class SplashScreen extends StatefulWidget {
+  const SplashScreen({super.key});
+
+  @override
+  State<SplashScreen> createState() => _SplashScreenState();
+}
+
+class _SplashScreenState extends State<SplashScreen> {
+  @override
+  void initState() {
+    super.initState();
+    nextAction();
+  }
+
+  void nextAction() async {
+    await Future.delayed(const Duration(seconds: 3), () {});
+    if (!mounted) return;
+    Navigator.pushNamedAndRemoveUntil(
+      context,
+      RouteNames.dashboard,
+      (route) => false,
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Container(
+        width: MediaQuery.of(context).size.width,
```

Comment:
I suggest using the newer mediaquery accessors - MediaQuery.sizeOf(context)

It won't affect much on this page, but it would be good to use it as an example for the community to follow.

The difference is that MediaQuery.of(context).size rebuilds the widget whenever any of the properties of mediaquery change, but MediaQuery.sizeOf(context) only rebuilds when the size changes

Reply:
resolved